sync2jira/upstream_issue.py

```diff
@@ -187,6 +187,13 @@ def github_issues(upstream, config):
         .get('github', {})\
         .get(upstream, {})
 
+    for key, expected in _filter.items():
+        if key == 'labels':
+            # We have to flatten the labels list to a comma-separated string
+            _filter[key] = ','.join(expected)
+        else:
+            _filter[key] = expected
```

Comment:
If we hit the `else` clause, the value of `_filter[key]` already _is_ `expected`, so there is no need to reassign it.  You should just omit the `else` clause.

Also, I don't think the `key` `'labels'` can appear in `_filter.items()` more than once, so there is no need to iterate over the dictionary, just look up the key and replace it:
```suggestion
    _filter['labels'] = ','.join(expected)
```

Reply:
Right, applied your suggestion (with a check since the `'labels'` filter is optinal)